src/textual/widgets/_data_table.py

```diff
@@ -131,6 +131,8 @@ class ColumnKey(StringKey):
 class CellKey(NamedTuple):
     """A unique identifier for a cell in the DataTable.
 
+    A cell key is a `(row_key, column_key)` tuple.
```

Comment:
As per your main comment, I think if it were me I'd be tempted to do both: make a note to this effect here, but also add docstrings to the two properties as well.

Reply:
Done, WDYT?

Reply:
I think it's perfect. :-)